agent/Cargo.toml

```diff
@@ -13,15 +13,15 @@ default = ["tokio-uring"]
 anyhow = "1.0"
 bytes = "1.2"
 clap = { version = "4", features = ["cargo", "derive"] }
-crypto-auditing = { path = "../crypto-auditing" }
+crypto-auditing = { version = "0.1", path = "../crypto-auditing" }
```

Comment:
do I understand correctly that when one does `cargo build`, the `path` is used, but when one does `cargo package`, it looks it up at crates.io and will generate a `Cargo.toml` that will look it up at crates.io?

Reply:
Yes, that should.